src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+
+const ChatLog = (props) => {
+// props is going to an array of objects
+    const chatEntryArray = props.entries.map((chatEntry) => {
+        return (
+            <ChatEntry
+                key={chatEntry.id}
+                id = {chatEntry.id}
+                sender={chatEntry.sender}
+                body={chatEntry.body}
+                timeStamp={chatEntry.timeStamp}
+                liked={chatEntry.liked}
+                onUpdate={props.onUpdate}
+            />
+        )
+    })
+
+    return (
+        <>
+            <section>
+                <ul className="container">{chatEntryArray}</ul>
+            </section>
+        </>
+    )
+
+}
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(PropTypes.object).isRequired,
+    onUpdate: PropTypes.func
+}
```

Comment:
Because we know the shape of the `entries` data structure, we can validate nested data within it using the `shape` method:

```suggestion
Don't forget to add PropTypes to help validate our props and avoid future errors. 

```python
ChatLog.propTypes = {
  entries: PropTypes.arrayOf(
    PropTypes.shape({
      id: PropTypes.number.isRequired,
      sender: PropTypes.string.isRequired,
      body: PropTypes.string.isRequired,
      timeStamp: PropTypes.string.isRequired,
      liked: PropTypes.bool,
    })
  ).isRequired,
  onUpdate: PropTypes.func.isRequired,
};
```
```